tfjs-layers/src/layers/preprocessing/image_resizing_test.ts

```diff
@@ -86,8 +86,13 @@ describeMathCPUAndGPU('Resizing Layer', () => {
     // create a resizing layer with same shape as input
     const height = 64;
     const width = 32;
-    const numChannels = 3;
-    const inputTensor = randomNormal([height, width, numChannels]);
+    const numChannels = 1;
+    const rangeArr = [...Array(height * width).keys()];
+    const inputArr = [];
+    while(rangeArr.length) {
+      inputArr.push(rangeArr.splice(0, width));
+    }
+    const inputTensor = tensor(inputArr, [height, width, numChannels]);
```

Comment:
Nit: [`tf.range`](https://js.tensorflow.org/api/latest/#range) and [`tf.reshape`](https://js.tensorflow.org/api/latest/#reshape) can make this simpler.

```suggestion
    const rangeTensor = range(0, height * width);
    const inputTensor = reshape(rangeTensor, [height, width, numChannels]);
```

Reply:
It looks like this applies to several other tests as well. Sorry I didn't catch this in my first review.